core/src/main/java/org/apache/carbondata/core/util/SessionParams.java

```diff
@@ -155,6 +141,12 @@ private boolean validateKeyValue(String key, String value) throws InvalidConfigu
       case CARBON_PUSH_ROW_FILTERS_FOR_VECTOR:
       case CARBON_ENABLE_INDEX_SERVER:
       case CARBON_QUERY_STAGE_INPUT:
+      case CARBON_LOAD_SI_REPAIR:
+        isValid = CarbonUtil.validateBoolean(value);
```

Comment:
CARBON_LOAD_SI_REPAIR switch case can be just a fall-through without case block. Same like above switch cases having same case block.

Reply:
removed